en_ewt-ud-train.conllu

```diff
@@ -177801,15 +177801,15 @@
 16	he	he	PRON	PRP	Case=Nom|Gender=Masc|Number=Sing|Person=3|PronType=Prs	19	nsubj	19:nsubj|22:nsubj	_
 17	was	be	AUX	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	19	aux	19:aux	_
 18	actually	actually	ADV	RB	_	19	advmod	19:advmod	_
-19	walkin	walkin	VERB	VBG	Tense=Pres|VerbForm=Part	13	parataxis	13:parataxis	_
+19	walkin	walk	VERB	VBG	Tense=Pres|VerbForm=Part	13	parataxis	13:parataxis	_
```

Comment:
`Typo=Yes`, `CorrectForm=walking`

Reply:
Yes, but nothing to do with improving the lemmas of comparatives and superlatives!

Reply:
I did fix it while I was here.  Would be happy to not have this spiral too far out of hand, though!